Ticker.history: history's index is not of type DateTimeIndex when dividend and split tables are empty

**Issue**:

Ticker.history (line 295):
>       df.index = df.index.tz_localize("UTC").tz_convert(data["chart"]["result"][0]["meta"]["exchangeTimezoneName"])
E       AttributeError: 'Index' object has no attribute 'tz_localize'

**Reason**: 

During the concatenation of a proper price table with empty dividend and split tables, the price table's index has changed from _pandas.DatetimeIndex_ to _pandas.Index_ because the empty tables had this index type.

**Proposed solution**: 

Modify _parse_actions_ (in utils.py) to initialise the dividend and split tables with : 
`dividends = _pd.DataFrame(columns=["Dividends"], index=_pd.DatetimeIndex([]))`
 `splits = _pd.DataFrame(columns=["Stock Splits"], index=_pd.DatetimeIndex([]))`


Local modules' versions:

finance : 0.1.69
pandas : 1.4.0
python : 3.10

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,4 @@
-pandas>=0.24.0
+pandas>=0.24.0,<=1.3.5
 numpy>=1.16.5
 requests>=2.26
 multitasking>=0.0.7

--- a/setup.py
+++ b/setup.py
@@ -61,7 +61,7 @@ setup(
     platforms=['any'],
     keywords='pandas, yahoo finance, pandas datareader',
     packages=find_packages(exclude=['contrib', 'docs', 'tests', 'examples']),
-    install_requires=['pandas>=0.24', 'numpy>=1.15',
+    install_requires=['pandas>=0.24.0,<=1.3.5', 'numpy>=1.15',
                       'requests>=2.26', 'multitasking>=0.0.7',
                       'lxml>=4.5.1'],
     entry_points={

